README.md

```diff
@@ -35,43 +46,65 @@ This project contains various building blocks for the Kubernetes Java developer
 
 
 ### Kubernetes Maven Plugin
-[![Maven Central](https://img.shields.io/maven-central/v/org.eclipse.jkube/kubernetes-maven-plugin.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:%22org.eclipse.jkube%22%20AND%20a:%22kubernetes-maven-plugin%22)
-[![Documentation](https://img.shields.io/badge/plugin-documentation-lightgrey)](https://www.eclipse.org/jkube/docs/kubernetes-maven-plugin)
-
-[![Sample Demo KMP](./kubernetes-maven-plugin/kmp.png)](https://asciinema.org/a/335724)
+- [![Maven Central](https://img.shields.io/maven-central/v/org.eclipse.jkube/kubernetes-maven-plugin.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:%22org.eclipse.jkube%22%20AND%20a:%22kubernetes-maven-plugin%22)
+- [![Documentation](https://img.shields.io/badge/plugin-documentation-lightgrey)](https://www.eclipse.org/jkube/docs/kubernetes-maven-plugin)
+- Add to project:
+  ```xml
+  <plugin>
+    <groupId>org.eclipse.jkube</groupId>
+    <artifactId>kubernetes-maven-plugin</artifactId>
+    <version>${jkube.version}</version>
+  </plugin>
+  ```
+- :tv: Watch 2-minute demo on YouTube:
```

Comment:
Maybe provide an example of what's next ?

```suggestion

    - Build, create and push the container image,  generate and apply the Kubernetes resources
      ```
      mvn package k8s:build k8s:push k8s:generate k8s:apply
      ```

- :tv: Watch 2-minute demo on YouTube:
```

Reply:
No I mean after each plugins, provide some example of how to use the plugin.

Reply:
I've added it as per your suggestion